src/todo/todo.lua

```diff
@@ -199,11 +208,12 @@ function todo.get_task_from_add_frame(frame)
     return task, should_add_to_top
 end
 
-function todo.create_task(text, assignee)
+function todo.create_task(text, assignee, creator)
```

Comment:
This could also automatically set the updated_by as it will be the same for creating tasks.

Reply:
Right. Missed this one. Fixed.